index.html

```diff
@@ -148,6 +148,30 @@ <h2>
         method.
         </li>
       </ul>
+      <p>
+        A payment method defines:
+      </p>
+      <dl>
+        <dt>
+          <dfn>Payment Method additional data type</dfn>
+        </dt>
+        <dd>
+          An IDL type that defines the data the <a>payment method</a> expects
+          to receive in the {{PaymentMethodData/data}} member of the
+          {{PaymentMethodData}}. If not specified for a given payment method,
+          no conversion to IDL is done and the payment method will receive
+          {{PaymentMethodData/data}} as JSON.
+        </dd>
+        <dt>
+          <dfn>Steps to validate payment method data</dfn>
```

Comment:
Hmmm... this doesn't actually define any steps? Maybe we don't need this definition?